tardis/simulation/convergence_plot.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class ConvergencePlot:
+    @staticmethod
```

Comment:
if it's a static method, you don't need `self` as an argument.

Reply:
@sonachitchyan removed that in the latest PR. Please check

Reply:
Did you remove the self from the calls in the simulation as well?  What was the error?

Reply:
@Rodot- I think now it's resolved. Please have a look!